### PR TITLE
testing AddStudent

### PR DESCRIPTION
**Pull request by jebonbright5**
- AddStudent button is temporarily on its own /path
- Student state now storing name and email correctly 
- addStudent method is not working to send fields to database

### CODE REVIEW

File: src/components/AddStudent.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    // Save student and close modal form
+    handleAdd = () => {
+        console.log("state" + JSON.stringify(this.state));
+        this.props.addStudent(this.state);
+        this.handleClose();
+    }
+
+    // add student
+    addstudent = (student) => {
```
Comment (ASMach) [inline src/components/AddStudent.js]: Where are you calling this function from? Is it related to line 52?

File: src/components/AddStudent.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    handleChangeEmail = (event) => {
+        this.setState({
+            email: event.target.value
+        });
+    }
+
+    // Save student and close modal form
+    handleAdd = () => {
+        console.log("state" + JSON.stringify(this.state));
+        this.props.addStudent(this.state);
```
Comment (ASMach) [inline src/components/AddStudent.js]: do you need to go through your props to call an arrow function that you already have in your component?

File: src/components/AddStudent.js (hunk 1 above)
Comment (jebonbright5) [reply src/components/AddStudent.js]: That was the intention although I have doubts about its correctness. I have been trying to follow other examples in the code provided as a guide. I believe SchedList.js is where I took that as an example from.

Comment (jebonbright5) [reply src/components/AddStudent.js]: sorry, it was AddCourse that I took as a reference. Not SchedList.

**Pull request merged**